en/docs/reference/pricing.md

```diff
@@ -2,229 +2,142 @@
-- The web application proceeds to make **1 API call to the [Reservations API](#reservationsAPI)**. This is counted as 1 step.
-- The Reservation API then makes **1 API call to the [Bookings API](#bookingsAPI)**. This is counted as **1 step**. 
-- The bookings API makes **1 API call to the Booking Database** to update it with the booking details. This is counted as **1 step**.
-- Upon receiving the booking confirmation, the Reservation API makes **1 call to the [Twilio API](#twillioAPI)** to send an SMS confirmation to the customer. This is counted as **1 step**.
-
-- **Assuming the entire execution time for the reservation process takes 1200ms (1.2s)** , in addition to the above steps,  the total number of extra computation units consumed can be calculated as follows using ***floor((t)/500)) where t is the execution time***: 
-
-          floor(1200/500) = 2 units = 2 steps
-
-Therefore the total number of steps for the reservation process is: 
-
-           4 steps + 2 steps = 6 steps
+## Plans
 
-We can calculate the **total number of steps for searching and reserving a flight** as follows:
+Choreo currently supports the following payment plans:
 
-- **Search process**: Single execution of a search would cost a total of **2 steps**. 
-- **Reservation process**: Single execution of reservation process would cost a total of **6 steps**. 
-- **Total**: A single execution of the search and reservation process of a flight would cost an organization a total of **8 steps in total**.
+**Free**
 
-!!! note
-    The above cost calculation is dependent on the above architecture and design of the APIs.
-<br/>
+You can enjoy the free tier with limitations and then move on to the Pay-As-You-Go model if you want to deploy over 5 components.
 
-### Example 2: REST API Proxy
-An organization exposes a flight tracking API as a managed API on Choreo. This system involves several APIs and databases. The scenario and the APIs in the system are described below:
+Choreo allocates up to a maximum of 2x vCPUs and 4 GB RAM infrastructure resources to your component workloads.
 
-#### Scenario
+**Pay As You Go**
 
-To search for real-time flight data, a client enters a search criteria, and searches through the web application. Upon receiving the request, the client application (web application) makes a call to the [Flights Status API](#flightStatusAPI), a managed API on Choreo, to search for flight data. Choreo then invokes the backend API, the [Real-Time Flight API](#realtimeFlightAPI). The backend API then invokes a database to get the information and passes it to the client.
+This plan requires you to pay $150 per component (except for the first five components) and pay for any additional infrastructure costs incurred (i.e., if the infrastructure cost is over $100). It does not require you to pay an annual fee. You can deploy up to 25 components (i.e., any combination of component types). 
```

Comment:
Shouldn't we mention that this $150 per component is monthly basis?

Reply:
Changed as follows:

```
This plan requires you to pay the following:
- $150 per component (except for the first five components) on a monthly basis
- Any additional infrastructure costs incurred (i.e., if the infrastructure cost is over $100). 

This plan does not require you to pay an annual fee. You can deploy up to 25 components (i.e., any combination of component types). 
```